setup.sql

```diff
@@ -0,0 +1,149 @@
+CREATE DATABASE novels;
+
+CREATE USER 'Alexandre'@'localhost' IDENTIFIED BY 'DUM@s!';
+
+GRANT ALL ON novels.* TO 'Alexandre'@'localhost';
+
+USE novels;
+
+CREATE TABLE authors (
+  id INT auto_increment,
+  nameFirst VARCHAR(255) NOT NULL,
+  nameLast VARCHAR(255) NOT NULL,
+  createdAt DATETIME DEFAULT NOW(),
+  updatedAt DATETIME DEFAULT NOW() ON UPDATE NOW(),
+  deletedAt DATETIME,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE novels (
+  id INT auto_increment,
+  title VARCHAR(255) NOT NULL,
+  authorId INT,
+  createdAt DATETIME DEFAULT NOW(),
+  updatedAt DATETIME DEFAULT NOW() ON UPDATE NOW(),
+  deletedAt DATETIME,
+  PRIMARY KEY(id),
+  FOREIGN KEY (authorId) REFERENCES authors(id)
+);
+
+CREATE TABLE genres (
+  id INT auto_increment,
+  genre VARCHAR(255) NOT NULL,
+  createdAt DATETIME DEFAULT NOW(),
+  updatedAt DATETIME DEFAULT NOW() ON UPDATE NOW(),
+  deletedAt DATETIME,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE booktype (
```

Comment:
`novelsGenres` would be a better name. Generally when you have a linking table naming it with the names of the tables it joins is helpful to know what it is